### PR TITLE
[ISSUE #283]Translation for "Deployment-RocketMQ Prometheus Exporter" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #283: Translation for "Deployment-RocketMQ Prometheus Exporter" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "Deployment-RocketMQ Prometheus Exporter" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #283 

## Brief changelog

- Translation for "Deployment-RocketMQ Prometheus Exporter" Section in the v4.x & 5.0 Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md [hunk 1]
```diff
@@ -1,355 +1,282 @@
...
-| rocketmq_consumer_message_size          | 订阅组在一个broker组上消费消息大小的TPS                      |
-| rocketmq_send_back_nums                 | 订阅组在一个broker组上消费失败，写入重试消息的次数           |
-| rocketmq_group_get_latency_by_storetime | 消费组消费延时，exporter get到消息后与当前时间相减           |
+| rocketmq_group_diff                     | Consumer group message accumulation message count            |
+| rocketmq_group_retrydiff                | Consumer group retry queue accumulation message count        |
+| rocketmq_group_dlqdiff                  | Consumer group dead letter queue accumulation message count  |
+| rocketmq_group_count                    | Number of consumers in the consumer group                    |
+| rocketmq_client_consume_fail_msg_count  | Number of times consumers in the consumer group have failed to consume in the past 1 hour |
+| rocketmq_client_consume_fail_msg_tps    | Consumer group consumer failure TPS                          |
+| rocketmq_client_consume_ok_msg_tps      | Consumer group consumer success TPS                          |
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md]: TPS of consumer consumption success

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md [hunk 2]
```diff
@@ -1,355 +1,282 @@
...
-| rocketmq_group_consume_total_offset     | 订阅组当前消费的Offset（对rocketmq_consumer_offset按broker聚合） |
-| rocketmq_consumer_message_size          | 订阅组在一个broker组上消费消息大小的TPS                      |
-| rocketmq_send_back_nums                 | 订阅组在一个broker组上消费失败，写入重试消息的次数           |
-| rocketmq_group_get_latency_by_storetime | 消费组消费延时，exporter get到消息后与当前时间相减           |
+| rocketmq_group_diff                     | Consumer group message accumulation message count            |
+| rocketmq_group_retrydiff                | Consumer group retry queue accumulation message count        |
+| rocketmq_group_dlqdiff                  | Consumer group dead letter queue accumulation message count  |
+| rocketmq_group_count                    | Number of consumers in the consumer group                    |
+| rocketmq_client_consume_fail_msg_count  | Number of times consumers in the consumer group have failed to consume in the past 1 hour |
+| rocketmq_client_consume_fail_msg_tps    | Consumer group consumer failure TPS                          |
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md]: TPS of consumer consumption failure

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md [hunk 3]
```diff
@@ -1,355 +1,282 @@
...
+### Start exporter application
 
-### 按照 promethus 官网配置启动
+### Start prometheus according to the configuration on its official website
 
-配置 promethus 的 static_config: -targets 为 exporter 的启动 IP 和端口,如: localhost:5557
+Configure prometheus's static_config: -targets to the exporter's starting IP and port, such as: localhost:5557.
 
-### 访问 promethus 页面
+### Access the prometheus page
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md]: Better capitalized prometheus into Prometheus, the same goes for others in the doc. Please check other prometheus usages again. Thx.

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md [hunk 4]
```diff
@@ -1,355 +1,282 @@
...
 
-本地启动默认为: localhost:9090 ,则可对收集到的指标值进行查看,如下图所示:
+If the localhost starts at the default localhost:9090, you can view the collected metric values, as shown in the following figure:
 
 <img src="https://tva1.sinaimg.cn/large/e6c9d24egy1h4l66aaa7tj215y0bzwi0.jpg"></img>
 
-
 :::tip
-为了达到更好的可视化效果,观察指标值变化趋势, promethus 搭配 grafana 效果更佳哦!
+For better visualization effects and to observe the trend of metric value changes, prometheus is better used with grafana!
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md]: prometheus -> Prometheus
grafana -> Grafana

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]

**Comment by mxsm** [comment]
@tsunghanjacktsai  I have fixed it and re-submit it. PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**